snuba_sdk/legacy.py

```diff
@@ -105,6 +105,8 @@ def parse_exp(value: Any) -> Any:
         return parse_scalar(value)
 
     alias = value[2] if len(value) > 2 else None
+    if alias and alias.startswith("`") and alias.endswith("`"):
+        alias = alias[1:-1]
```

Comment:
IS it harder to actually fix this at the source ?

Reply:
I don't know about harder. It was certainly faster to fix it here, rather than hunting through all of Sentry to try and find every use case of this.